python/cudf/cudf/tests/test_datetime.py

```diff
@@ -1452,7 +1452,14 @@ def test_is_month_start(data, dtype):
 date_range_test_periods = [1, 10, 100]
 date_range_test_freq = [
     {"months": 3, "years": 1},
-    {"hours": 10, "days": 57, "nanoseconds": 3},
+    pytest.param(
+        {"hours": 10, "days": 57, "nanoseconds": 3},
+        marks=pytest.mark.xfail(
+            True,
+            reason="Pandas ignoring nanoseconds component. "
+            "https://github.com/pandas-dev/pandas/issues/44393",
+        ),
+    ),
```

Comment:
@isVoid I'm not sure how/why we were passing these tests before, but it looked to me like we were broken in a similar way to Pandas.